bundles/binding/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/state/PowerMaxPanelType.java

```diff
@@ -0,0 +1,197 @@
+/**
+ * Copyright (c) 2010-2016, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.powermax.internal.state;
+
+/**
+ * Used to store main characteristics of each Visonic alarm panel type in an ENUM
```

Comment:
Could you add links to source reference documentation in comments throughout the binding so other contributors and reviewers could understand the correlation between your code and the documented protocols involved?